code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -323,9 +323,7 @@
 	var/teleporting = FALSE
 
 /obj/item/warp_cube/attack_self(mob/user)
-	var/turf/current_location = get_turf(user)
```

Comment:
does any other ruin/area rely on this to prevent teleporting

Reply:
it was added by king goat pr so probbablably not